src/main/java/com/nutomic/syncthingandroid/syncthing/AppConfigReceiver.java

```diff
@@ -0,0 +1,87 @@
+package com.nutomic.syncthingandroid.syncthing;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.activities.MainActivity;
+
+/**
+ * Broadcast-receiver to control and configure SyncThing remotely
+ *
+ * Created by sqrt-1764 on 25.03.16.
+ */
+public class AppConfigReceiver extends BroadcastReceiver {
+    private static final int ID_NOTIFICATION_BACKGROUND_ACTIVE = 2;
+
+    public static final String ACTION_START = "com.nutomic.syncthingandroid.action.START";
+
+    public static final String ACTION_STOP  = "com.nutomic.syncthingandroid.action.STOP";
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        switch (intent.getAction()) {
+            case ACTION_START:
+                SyncthingHelper.startSynchthingService(context);
+                break;
+
+            case ACTION_STOP:
+                SyncthingHelper.stopSynchtingService(context);
+                break;
+        }
+    }
+
+
+    private static class SyncthingHelper {
```

Comment:
I still don't think we need this helper class. Just put the code from `startSyncthingService()` directly into the switch, and put `stopSynchtingService()` directly in AppConfigReceiver. We can always change this later if needed.